components/VoteParticipation/VoteParticipation.tsx

```diff
@@ -7,7 +7,13 @@ import { QUERIES, BREAKPOINTS } from "constants/breakpoints";
 import { useWindowSize, useIntersectionObserver } from "hooks";
 import Image from "next/image";
 
-const VoteParticipation = () => {
+interface Props {
+  apy: string;
+}
+
+// If API errors, show a default guess APY.
+export const DEFAULT_APY = "30.1";
```

Comment:
we might want to be able to override this with an env, and default to 30.1 if env is not defined. it will just be annoying to have to change this with code